NGPIris/cli/base.py

```diff
@@ -22,13 +22,19 @@
 @click.pass_context
 def root(ctx, endpoint, access_key_id, access_key, bucket, credentials, password, logfile):
     """NGP intelligence and repository interface software"""
-    c = preproc.read_credentials(credentials)
+    if credentials:
+        c = preproc.read_credentials(credentials)
+    else:
+        c = {}
 
     ep = endpoint if endpoint else c.get('endpoint','')
     aid = access_key_id if access_key_id else c.get('aws_access_key_id','') 
     key = access_key if access_key else c.get('aws_secret_access_key','')
     pw = password if password else c.get('ngpi_password', '')
 
+    if not all([ep, aid, key]):
+        print("Missing essential HCP credentials. Please provide them manually or via a credentials file.")
```

Comment:
This is integrated in `read_credentials`. I should probably write that out a bit better